rosidl_runtime_py/rosidl_runtime_py/convert.py

```diff
@@ -33,13 +59,16 @@ def __represent_ordereddict(dumper, data):
     return yaml.nodes.MappingNode(u'tag:yaml.org,2002:map', items)
 
 
-def message_to_yaml(msg: Any, truncate_length: int = None) -> str:
+def message_to_yaml(msg: Any, truncate_length: int = None,
+                    no_arr: bool = False, no_str: bool = False) -> str:
```

Comment:
This doesn't follow the style guide. Please wrap after the open parenthesis and indent all arguments one level (without any alignment).

It would be good to also force the keyword arguments to be passed as such (using a `*` between the positional and keyword arguments).

Same below.

Reply:
@jacobperron Friendly ping.

Reply:
See https://github.com/ros2/rosidl_python/pull/80